frontend.rst

```diff
@@ -84,26 +107,44 @@ Full API
 
 * `Full API`_
 
+.. _frontend-asset-mapper:
+
+Asset Mapper
+------------
+
+Asset Mapper is an alternative to Webpack Encore that runs entirely in PHP
+without any complex build steps. It leverages the ``importmap`` feature of
+your browser, which is available in all browsers thanks to a polyfill.
+Asset Mapper is currently experimental.
+
+* :doc:`Install AssetMapper </frontend/asset_mapper/installation>`
+* :doc:`Using AssetMapper </frontend/asset_mapper/usage>`
+* :doc:`Handling CSS </frontend/asset_mapper/css>`
+* :doc:`FAQ & Common Issues </frontend/asset_mapper/faq>`
+
 Symfony UX Components
 ---------------------
 
+* :doc:`/frontend/ux`
+
 .. include:: /frontend/_ux-libraries.rst.inc
 
 Other Front-End Articles
 ------------------------
 
+* :doc:`/frontend/create_ux_bundle`
+* :doc:`/frontend/custom_version_strategy`
+
 .. toctree::
     :hidden:
     :glob:
 
     frontend/encore/installation
     frontend/encore/simple-example
     frontend/encore/*
-
-.. toctree::
-    :maxdepth: 1
-    :glob:
-
+    frontend/asset_mapper/installation
+    frontend/asset_mapper/usage
+    frontend/asset_mapper/*
     frontend/*
```

Comment:
The bottom of https://symfony.com/doc/current/frontend.html is currently "wonky".  I think the intention was for the the first, `:hidden` `toctree` to "load" all of the `encore` stuff into the `toctree`... and then this bottom `toctree` would only load anything "remaining". In practice, it would render everything.

So, I reduced this to just ONE `toctree`, which is `:hidden:`. The 2 documents that are outside of `encore/*` were added manually under the `Other Front-End Articles` section.

Reply:
That IS a fun fact, thank you!